cumulus/parachains/runtimes/collectives/collectives-westend/src/fellowship/tracks.rs

```diff
@@ -114,211 +116,213 @@ pub struct TracksInfo;
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(50),
 						ceil: Perbill::from_percent(100),
 					},
-					min_support: pallet_referenda::Curve::LinearDecreasing {
+					min_support: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(0),
 						ceil: Perbill::from_percent(100),
 					},
 				},
-			),
-			(
-				tracks::ARCHITECTS_ADEPT,
-				pallet_referenda::TrackInfo {
-					name: "architects adept",
+			},
+			Track {
+				id: tracks::ARCHITECTS_ADEPT,
+				info: TrackInfo {
+					name: s("architects adept"),
 					max_deciding: 10,
 					decision_deposit: 5 * DOLLARS,
 					prepare_period: 30 * MINUTES,
 					decision_period: 1 * DAYS,
 					confirm_period: 30 * MINUTES,
 					min_enactment_period: 5 * MINUTES,
-					min_approval: pallet_referenda::Curve::LinearDecreasing {
+					min_approval: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(50),
 						ceil: Perbill::from_percent(100),
 					},
-					min_support: pallet_referenda::Curve::LinearDecreasing {
+					min_support: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(0),
 						ceil: Perbill::from_percent(100),
 					},
 				},
-			),
-			(
-				tracks::GRAND_ARCHITECTS,
-				pallet_referenda::TrackInfo {
-					name: "grand architects",
+			},
+			Track {
+				id: tracks::GRAND_ARCHITECTS,
+				info: TrackInfo {
+					name: s("grand architects"),
 					max_deciding: 10,
 					decision_deposit: 5 * DOLLARS,
 					prepare_period: 30 * MINUTES,
 					decision_period: 1 * DAYS,
 					confirm_period: 30 * MINUTES,
 					min_enactment_period: 5 * MINUTES,
-					min_approval: pallet_referenda::Curve::LinearDecreasing {
+					min_approval: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(50),
 						ceil: Perbill::from_percent(100),
 					},
-					min_support: pallet_referenda::Curve::LinearDecreasing {
+					min_support: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(0),
 						ceil: Perbill::from_percent(100),
 					},
 				},
-			),
-			(
-				tracks::MASTERS,
-				pallet_referenda::TrackInfo {
-					name: "masters",
+			},
+			Track {
+				id: tracks::MASTERS,
+				info: TrackInfo {
+					name: s("masters"),
 					max_deciding: 10,
 					decision_deposit: 5 * DOLLARS,
 					prepare_period: 30 * MINUTES,
 					decision_period: 1 * DAYS,
 					confirm_period: 30 * MINUTES,
 					min_enactment_period: 5 * MINUTES,
-					min_approval: pallet_referenda::Curve::LinearDecreasing {
+					min_approval: Curve::LinearDecreasing {
 						length: Perbill::from_percent(100),
 						floor: Perbill::from_percent(50),
 						ceil: Perbill::from_percent(100),
 					},
-					min_support: pallet_referenda::Curve::LinearDecreasing {
+					min_support: Curve::LinearDecreasing {
```

Comment:
Just as a tip: there are a lot of noisy changes in this MR - it can scare people of to take a look.

Reply:
Will make it less noisy, thanks for having a look :)